# project1
Analysis of RNA-seq data to identify differentially expressed genes between parental sensitive tumour cells and derived resistant tumour cells. A gene ontology analysis was also performed to identify the underlying biology.
Basic data processing, quality control, adapter trimming and  salmon quantification were carried out in a high performance computing environment using Linux and shell scripts.
Downstream analysis on salmon quantification files was done in Jupyter notebook using R. The DESeq2 function was used in differential gene expression calculation.  The topGO package was employed in the gene ontology analysis.  
